src/themes/darkTheme.js

```diff
@@ -3,11 +3,12 @@ const darkTheme = {
     primary: '#dcb042',
     secondary: '#cb042',
     background: '#121212',
-    color: '#eee',
+    color: '#fff',
     text: '#fef',
     darkGrey: '#a9a9a9',
     lightGrey: '#bebdbd',
```

Comment:
it's better to name theme "variables" what they are used for not what specific color they are, but it does make sense if it's a global color that never changes as it seems to be here